### PR TITLE
User statistics screen

### PR DESCRIPTION
**Pull request by NinaWie**
* Status names are now Gold Pro and Gold Legend
* cutoffs at 10, 50, 100, 500, 1000
* per-location statistic max at 25
* per location percentage max at 50

### CODE REVIEW

File: PennyMe/ViewController.swift [hunk 1]
```diff
@@ -469,6 +510,17 @@ class ViewController: UIViewController, UITextFieldDelegate, UIGestureRecognizer
                         // re-add annotations here
                         self?.isVisible = [:]
                         self?.addAnnotationsIteratively()
+                        
+                        // count how many machines we have in total
+                        totalMachines = 0
+                        machinesByArea = [:]
+                        for machine in self!.artworks {
+                            if machine.machineStatus != "retired" {
```
Comment (jannisborn) [inline PennyMe/ViewController.swift:518]: This means that we count "out-of-order" machines. Fine for me

File: PennyMe/StatisticsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+        
+        numberLabel.font = UIFont.systemFont(ofSize: 60, weight: .bold)
+        numberLabel.shadowColor = UIColor.gray // Shadow color
+        numberLabel.shadowOffset = CGSize(width: 2, height: 2)
+        numberLabel.textColor = color
+        numberLabel.text = "\(visitedCount)"
+        
+        totalMachinesLabel.numberOfLines = 0
+        totalMachinesLabel.text = "machines visited"
+        totalLabel.text = "/ \(totalMachines)"
```
Comment (jannisborn) [inline PennyMe/StatisticsViewController.swift:114]: Out of curiosity, why do we have access to the `totalMachines` variable here even though it is only defined in the ViewController?

**Review by jannisborn — APPROVED** [review]
Amazing job, really cool new feature :) 
A general thought -- currently we are not transparent to the users regarding the thresholds for the levels. They only see the threshold for the next level once they reach a new level. 
Should we make this more transparent? We could make the crown images clickable and display an info text when a user taps on it?

File: PennyMe/ViewController.swift (hunk 1 above)
Comment (NinaWie) [reply PennyMe/ViewController.swift:518]: Yes good point, that was intended

**Pull request merged**